### DISCUSSION TITLE
Clicking user profile icon in Windows 10 start menu didn't bring the account options

### DISCUSSION THREAD

**Discussion by adxmque** [valinet/ExplorerPatcher#3401] [Q&A]
Hi, I would like to report a minor issue that I encounter with ExplorerPatcher, it seems like when I set my start menu to Windows 10 style, clicking the user profile icon at the start menu sidebar didn't bring any user account option (the usual "Change Account Settings, Sign Out & Lock" selection). Not that I can't live with it, but I find it is one of the minor touches that need to be taken care of to recreate the Windows 10 experience.

I can confirm it's a bug with ExplorerPatcher because I install it on two separate Windows 11 devices with similar settings, and both exhibit the same bug.

For your reference, here's my device specification:
Windows 11 Build - 22631.3737 (23H2)
ExplorerPatcher Build - 22621.3527.65.1

And here is a visual recording to further explain the bug:
https://github.com/valinet/ExplorerPatcher/assets/66345173/fd5bc3ad-6c41-4f78-90b9-cc85eb39b987

**Comment by pyrates999** [comment]
This was fixed in the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.5_6f6666b
